lib/pengine/unpack.c

```diff
@@ -3762,14 +3771,16 @@ unpack_rsc_op(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
         }
     }
 
-    /* If the executor reported an operation status of anything but done or
-     * error, consider that final. But for done or error, we know better whether
-     * it should be treated as a failure or not, because we know the expected
-     * result.
-     */
-    if(status == PCMK_EXEC_DONE || status == PCMK_EXEC_ERROR) {
-        status = determine_op_status(rsc, rc, target_rc, node, xml_op, on_fail, data_set);
-        pe_rsc_trace(rsc, "Remapped %s status to %d", task_key, status);
+    if (ignore_mask_probe_failure) {
+        crm_notice("Ignoring masked probe failure from resource %s on %s",
+                   rsc->id, node->details->uname);
```

Comment:
The word failure might scare users ... something like: Treating probe result '%s' for %s on %s as 'not running'

Reply:
Fixed.